_posts/brice_aminou/2017-03-07-Mobile-app-with-Laravel.md

```diff
@@ -13,4 +13,123 @@ tags:
 teaser:
     info: An easy way to create a mobile app from an existing Laravel application
 ---
-This tutorial describes an easy way to create a mobile app from your existing Laravel application.
+This tutorial describes an easy way to create a mobile app from your existing Laravel application. At the end of this tutorial, you will be able to create a hybrid app from an
+existing Laravel project. A hybrid app have many advantages if you are familiar with HTML, CSS and Javascript:
+
+* The app can be published on the Play Store (iOS also, but it is not part of this tutorial)
+* There is no API to build so only one source code to manage
+* Works on any platform
+* Can access the features of a mobile phone like a native app
+
+This tutorial is using:
+
+* [Apache with PHP 7.0 and MySQL](https://www.digitalocean.com/community/tutorials/how-to-install-linux-apache-mysql-php-lamp-stack-on-ubuntu-16-04) Find the right installation for your OS and your required PHP version
+* [Laravel 5.*](https://laravel.com/docs/5.4/installation)
+* [Android Studio with latest SDK](https://developer.android.com/studio/index.html)
+* Ubuntu 16.04
+* [Composer](https://bappa.info/2016/05/07/install-composer-in-ubuntu-16-04/)
+
+## Laravel Installation
+If you already have an existing project, you can jump directly to the next step.
+
+SSH to your host and make sure that you have the permissions to edit the directory where your website is going to be hosted. If you do not have the permissions,
+ask your server administrator to give you the permissions.
+
+Go to the Apache public directory (Usually /var/www/html, but might change depending on your apache setup)
+
+~~~BASH
+  cd /var/www/html
+~~~
+
+Create a new project using composer. If you do not have composer installed, I invite you to look at other solutions to create a new Laravel application [in their
+documentation](https://laravel.com/docs/5.4/installation).
+
+~~~BASH
+composer create-project --prefer-dist laravel/laravel blog
+cd blog
+~~~
+
+Change the permissions of the project folders
+
+~~~BASH
+sudo chmod –R 777 storage
+sudo chmod –R 777 bootstrap
+sudo chmod –R 777 public
+~~~
+
+Warning: The permissions of those directories are changed to 777 only for this tutorial. Do not use those permissions in production.
```

Comment:
Why, for the purpose of this tutorial are you using 777 ? Maybe add an explanation.